lib/nconf/stores/file.js

```diff
@@ -185,7 +179,16 @@ File.prototype.stringify = function (format) {
   }
 
   if (this.secure) {
-    data = this.keys.encrypt(data);
+    var self = this;
+    data = Object.keys(data).reduce(function (acc, key) {
+      var value = format.stringify(data[key]);
+      var iv = crypto.randomBytes(16);
+      var cipher = crypto.createCipheriv(self.secure.alg, self.secure.secret, iv);
+      var ciphertext = cipher.update(value, 'utf8', 'hex');
+      ciphertext += cipher.final('hex');
+      acc[key] = { alg: self.secure.alg, value: ciphertext, iv: iv.toString('hex') };
```

Comment:
You should also store the authentication tag when `alg` is a CCM, GCM or OCB cipher (and store the full 128 bits, not one of the shorter variants.)